product_contract/views/sale_order.xml

```diff
@@ -121,7 +121,7 @@
                     domain="[('contract_id','=',contract_id)]"
                     optional="hide"
                 />
-                <field name="recurrence_number" optional="hide" />
+                <field name="recurrence_number" optional="show" />
```

Comment:
Please don't change this.

Reply:
Default is 1 but if you install this module this is most probably to play with this number.
I'm ok with showing the interval too.
My point is that the price is key on a sale order, imo every factor of the price should clearly be displayed to the user. Else, the SO might become hard to understand or look buggy.
I can already hear my customers scream at me because the SO shows wrong numbers :sweat_smile:

Reply:
But as the module includes its own dialog for configuring this, what we don't want is that the list includes a lot of columns, because that's a bad practice, so please keep both hidden by default. If the user needs them, they can show both.